--- alsa-plugins-1.0.17/a52/pcm_a52.c.orig	2008-10-13 01:16:02.481709383 +0200
+++ alsa-plugins-1.0.17/a52/pcm_a52.c	2008-10-13 01:16:14.480884705 +0200
@@ -25,7 +25,7 @@
 #include <alsa/asoundlib.h>
 #include <alsa/pcm_external.h>
 #include <alsa/pcm_plugin.h>
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 
 struct a52_ctx {
 	snd_pcm_ioplug_t io;
--- alsa-plugins-1.0.17/rate-lavc/rate_lavcrate.c.orig	2008-10-13 01:25:37.942159294 +0200
+++ alsa-plugins-1.0.17/rate-lavc/rate_lavcrate.c	2008-10-13 01:25:50.941265895 +0200
@@ -19,7 +19,7 @@
 #include <stdio.h>
 #include <alsa/asoundlib.h>
 #include <alsa/pcm_rate.h>
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 #include "gcd.h"
 
 static int filter_size = 16;
--- alsa-plugins-1.0.17/configure.in.orig	2008-10-13 01:21:23.459649300 +0200
+++ alsa-plugins-1.0.17/configure.in	2008-10-13 01:21:35.458824622 +0200
@@ -62,7 +62,7 @@
 AC_SUBST(AVCODEC_LIBS)
 AC_CHECK_LIB([avcodec], [avcodec_open], [HAVE_AVCODEC=yes], [HAVE_AVCODEC=no])
 if test x$HAVE_AVCODEC = xyes; then
-  AC_CHECK_HEADER([ffmpeg/avcodec.h], [], [HAVE_AVCODEC=no])
+  AC_CHECK_HEADER([libavcodec/avcodec.h], [], [HAVE_AVCODEC=no])
 fi
 AM_CONDITIONAL(HAVE_AVCODEC, test x$HAVE_AVCODEC = xyes)
 CFLAGS="$CFLAGS_saved"
